projects/eval-adxl355-pmdz/src/platform/aducm3029/platform_src.mk

```diff
@@ -8,4 +8,6 @@ endif
 SRC_DIRS += $(PLATFORM_DRIVERS)
 SRC_DIRS += $(INCLUDE)
 
+SRCS += $(NO-OS)/util/no_os_alloc.c
```

Comment:
i don't think this should be added here.

Reply:
Moved here because the memory allocation is different compared to the rest of the platforms